core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,26 +2,26 @@
 Benchmark ZStandardCompressionCodec
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_372-b07 on Linux 5.15.0-1041-azure
+OpenJDK 64-Bit Server VM 17.0.8+7-LTS on Linux 5.15.0-1046-azure
 Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
 Benchmark ZStandardCompressionCodec:                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 --------------------------------------------------------------------------------------------------------------------------------------
-Compression 10000 times at level 1 without buffer pool            293            327          85          0.0       29283.2       1.0X
-Compression 10000 times at level 2 without buffer pool            322            324           2          0.0       32184.8       0.9X
-Compression 10000 times at level 3 without buffer pool            453            456           2          0.0       45285.1       0.6X
-Compression 10000 times at level 1 with buffer pool               171            173           1          0.1       17065.2       1.7X
-Compression 10000 times at level 2 with buffer pool               208            209           1          0.0       20786.5       1.4X
-Compression 10000 times at level 3 with buffer pool               334            335           2          0.0       33350.3       0.9X
+Compression 10000 times at level 1 without buffer pool           2800           2801           2          0.0      279995.2       1.0X
```

Comment:
From what I remember, the results of this microbenchmark are always unstable.

Reply:
Yes, this one has been on my todo list. Will try to identify the root cause.